examples/meta/src/neural_nets/feedforward_net_classification.sg

```diff
@@ -0,0 +1,45 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
+CSVFile f_feats_test("../../data/fm_test_real.dat")
+CSVFile f_labels_train("../../data/label_train_twoclass.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+BinaryLabels labels_train(f_labels_train)
+#![create_features]
+
+#![add_layers]
```

Comment:
@sanuj check the swig interface file (dont know how it is called, but you need to tell swig that a method returns a new object in the occresponding `.i` file)